### PR TITLE
Initial Routing Configuration

### PR DESCRIPTION
**Pull request by G-Arv**
So I wasn't too sure how to go about implementing URLs so I made some express routing within the CreateAccountContainer, SignedInContainer, and SignedOutContainer. I've got two questions about it:
1. Would this be the way to create URLs? I was looking up ideas online and this was the closest idea I had.
2. Has this already been implemented in some way? I don't see it in the routes folder
3. In the CreateAccountContainer, there's a few sign-in page routes that are the same, but I'm not too sure what the correct URI would be, so I left it as is. Would this be correct?

Let me know what you think.

### CODE REVIEW

File: src/client/scripts/views/CreateAccount/CreateAccountContainer.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { HousingSituationView } from '../CreateAccount/HousingSituationView.js';
 import { NeedHousingView } from '../CreateAccount/NeedHousingView.js';
 import { HaveHousingView } from '../CreateAccount/HaveHousingView.js';
 import { Events } from '../../Events.js';
+import express from 'express';
```
Comment (ashleybhandari) [inline src/client/scripts/views/CreateAccount/CreateAccountContainer.js:11]: Express routing is used for HTTP and responding to client requests. This method will not change the URL. Also, Isha's working on routing, you can see her work in the PR's. If you google "change url without reloading page" many options will show up. Try following some of those.

**Comment by G-Arv** [comment]
This should fix the page and also fixes a grammatical error in DatabasePouchDB.js that was giving rendering errors.

File: src/client/scripts/views/CreateAccount/CreateAccountContainer.js [hunk 2]
```diff
@@ -61,28 +61,31 @@ export class CreateAccountContainer {
         if (view === 'sign-in') {         // SignInView
             this.#viewContainer.appendChild(this.#signInViewElm);
             // URL TODO: url should be 'index.html/sign-in'
+            history.replaceState(null, "sign-in", "/index.html/sign-in");
```
Comment (ashleybhandari) [inline src/client/scripts/views/CreateAccount/CreateAccountContainer.js:64]: Much better, thank you. I'm getting an issue where, when I try navigating between Discover, Matches, and Settings, "index.html" keeps getting appended. I think you can look into replaceState vs pushState or something for that.
image

Comment (G-Arv) [reply src/client/scripts/views/CreateAccount/CreateAccountContainer.js:64]: I forgot to fix the SignedIn files - I just changed it. Lmk if there's an error

**Pull request merged**